### PR TITLE
Mdkessler patch 12

### PR DESCRIPTION
**Pull request by mdkessler**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

**Comment by Benjamin-Lee** [comment]
I have my concerns about this tip in general but I think your changes are a net positive for structure and clarity.

**Review by Benjamin-Lee — APPROVED** [review]

File: content/11.interpretation.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-Because deep models can be difficult to interpret intuitively, there is a temptation to anthropomorphize the models.
-We must resist this temptation.
-
-A common saying in statistics classes is "correlation doesn't imply causality".
-While we know that accurately predicting an outcome doesn't imply learning the causal mechanism, it can be easy to forget this lesson when the predictions are extremely accurate.
-A poignant example of this lesson is [@tag:predicting-pneumonia-mortality; @doi:10.1145/2783258.2788613].
-In this study, the authors evaluated the capacities of several models to predict the probability of death for patients admitted to an intensive care unit with pneumonia.
+Once we have trained an accurate deep learning model, we often want to use it to deduce relationships and inform scientific findings.
+However, in doing this, we need to be careful to correctly interpret the model's predictions.
+Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to anthropomorphize the models, and to overinterpret the predictions in indulgent and/or inaccurate ways.
```
Comment (Benjamin-Lee) [inline content/11.interpretation.md:5]: Note that this line the focus of #268, which takes offense at the use of "anthropomorphize" in this context. Maybe we should just remove the phrase and replace it with what you wrote:

```suggestion
Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to overinterpret the predictions in indulgent and/or inaccurate ways.
```

File: content/11.interpretation.md [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+The neural network had therefore also learned to make predictions according to this rule despite the fact that it has nothing to do with causality or mechanism.
 Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.
 
 To trust deep learning models, we must combine knowledge of the training data ([Tip 4](#know-your-problem)) with inspection of the model ([Tip 8](#blackbox)).
-To move beyond fitting predictive models toward building understanding and deducing scientific conclusions, probe data domains where your model succeeds and contrast them with domains where your model fails in order to identify your model's internal logic, taking care to avoid overinterpreting or anthropomorphizing the model.
+To move beyond fitting predictive models and towards the building of an understanding that can inform scientific deduction, we suggest working to disanetangle a model's internal logic by comparing data domains where your model succeeds to those in which they fail. And again, take care not to overinterpret or anthropomorphize your models.
```
Comment (cgreene) [inline content/11.interpretation.md:17]:
```suggestion
To move beyond fitting predictive models and towards the building of an understanding that can inform scientific deduction, we suggest working to disentangle a model's internal logic by comparing data domains where models succeeds to those in which they fail.
And again, take care not to overinterpret or anthropomorphize models.
```

Comment (cgreene) [reply content/11.interpretation.md:17]: Last like says anthropomorphize again, but fixed the other elements

**Review by cgreene — APPROVED** [review]
A few suggestions - looks good 🚀

File: content/11.interpretation.md [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
+The neural network had therefore also learned to make predictions according to this rule despite the fact that it has nothing to do with causality or mechanism.
 Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.
 
 To trust deep learning models, we must combine knowledge of the training data ([Tip 4](#know-your-problem)) with inspection of the model ([Tip 8](#blackbox)).
-To move beyond fitting predictive models toward building understanding and deducing scientific conclusions, probe data domains where your model succeeds and contrast them with domains where your model fails in order to identify your model's internal logic, taking care to avoid overinterpreting or anthropomorphizing the model.
+To move beyond fitting predictive models and towards the building of an understanding that can inform scientific deduction, we suggest working to disentangle a model's internal logic by comparing data domains where models succeeds to those in which they fail.
+And again, take care not to overinterpret or anthropomorphize models.
```
Comment (Benjamin-Lee) [inline content/11.interpretation.md:18]:
```suggestion
By doing so, we can avoid overinterpreting models and view them for what they are: complex statistical models trained on high dimensional data.
```

**Comment by Benjamin-Lee** [comment]
@cgreene What do you think about this concluding sentence? I tried to reword it to avoid using the term "anthropomorphize" which seemed to be confusing

File: content/11.interpretation.md (hunk 3 above)
Comment (mdkessler) [reply content/11.interpretation.md:18]: @Benjamin-Lee I like your change.

Here's another alternative for consideration...just to throw it out there in case any of it appeals to anyone.

```suggestion
By doing so, we can avoid over interpreting these highly complex and frequently caveated statistical models.
```

Comment (cgreene) [reply content/11.interpretation.md:18]: I think Ben's works well. Shall we go with that?
